<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_bestellung</class>
 <widget class="QDialog" name="Dialog_bestellung">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>313</width>
     <height>327</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="1">
     <widget class="QRadioButton" name="radioButton_LT_unbekannt">
      <property name="text">
       <string>unbekannt</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_artikel_id">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Artikel-ID</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Artikel-Nr.</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_ve">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_lieferant">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_artikel_bez">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QSpinBox" name="spinBox_menge">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>99999999</number>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>VE</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Lieferant</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_artikel_nr">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Artikel-Bez.</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Menge</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QRadioButton" name="radioButton_LT_bekannt">
      <property name="text">
       <string>bekannt</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>311</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_ok">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_cancel">
      <property name="text">
       <string>Abbrechen</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>331</x>
     <y>261</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Kommentar:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_kommentar"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>331</x>
     <y>11</y>
     <width>250</width>
     <height>190</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Liefertermin:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCalendarWidget" name="calendarWidget_LT"/>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_artikel_id</tabstop>
  <tabstop>lineEdit_artikel_nr</tabstop>
  <tabstop>lineEdit_artikel_bez</tabstop>
  <tabstop>lineEdit_lieferant</tabstop>
  <tabstop>lineEdit_ve</tabstop>
  <tabstop>spinBox_menge</tabstop>
  <tabstop>radioButton_LT_unbekannt</tabstop>
  <tabstop>radioButton_LT_bekannt</tabstop>
  <tabstop>pushButton_ok</tabstop>
  <tabstop>pushButton_cancel</tabstop>
  <tabstop>calendarWidget_LT</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
